map_elites/environments.py

```diff
@@ -1,11 +1,15 @@
 import math
 import string
+from abc import ABC, abstractmethod
 
 import numpy as np
 from numpy import array
 
 from map_elites.map_elites import Genotype, Phenotype
```

Comment:
There was a problem of circular import when I tried to run `elm.py` from the previous directory. Was using Python 3.9.
Also, judging from the next two lines, are you running something from this directory instead?